MapboxCoreNavigation/Constants.swift

```diff
@@ -55,7 +50,7 @@ public let RouteControllerProgressDidChange = Notification.Name(MBRouteControlle
 /**
  Emitted when the alert level changes. This indicates the user should be notified about the upcoming maneuver.
```

Comment:
Since alert levels are no more, describe this notification in terms of the user passing an ideal point at which to announce an upcoming maneuver.